app/lib/audit_helper.rb

```diff
@@ -0,0 +1,9 @@
+module AuditHelper
+  def audit(actor)
+    impersonator = actor.try(:impersonator)
+
+    Audited.audit_class.as_user(impersonator || actor) do
```

Comment:
If there is an impersonator, do we want to audit this as something like `staff@digital.education.gov.uk on behalf of user@scitt.co.uk`?

Reply:
The audit itself can only be associated with one user and I think it's safer if this user is the support one.